docs/API.md

```diff
@@ -459,22 +476,23 @@ Example response:
 ```
 
 
-#### `"result"`
+##### `"result"`
```

Comment:
I think this should be on level 4 (`####`) not 5 since it is a subheader of `### API method: system_versions` on level 3.